### DISCUSSION TITLE
Cannot obtain result from activity_id "VolMIP"

### DISCUSSION THREAD

**Discussion by yxing6** [intake/intake-esm#529] [Q&A] [answered]
### Description

Cannot find results when filter with `activity_id="VolMIP"` and/or `experiment_id="volc-pinatubo-full"`.   
While these experiments can be found on the CMIP6 search webpage https://esgf-node.llnl.gov/search/cmip6/, they do not seem to exist in intake-esm's database.

### What I Did

```
import intake
import intake_esm

url = "https://storage.googleapis.com/cmip6/pangeo-cmip6.json"
dataframe = intake.open_esm_datastore(url)

# dataframe.df.head()        # works fine

model = dataframe.search(activity_id="VolMIP")
model
```
> <pangeo-cmip6 catalog with 0 dataset(s) from 0 asset(s)>

I also verified that `VolMIP` and `volc-pinatubo-full` are indeed not included in return result of `open_esm_datastore(url`)`
```
activity = dataframe.unique(["dataframe"])
"VolMIP" in activity
```
> False
```
experiment = dataframe.unique(["experiment_id"])
"volc-pinatubo-full" in experiment
```
> False

### Version information:

<details>

INSTALLED VERSIONS
------------------

cftime: 1.6.0
dask: 2021.04.0
fastprogress: 1.0.2
fsspec: 2022.5.0
gcsfs: 0.6.2
intake: 0.6.5
intake_esm: 2021.8.17
netCDF4: 1.5.7
pandas: 1.2.4
requests: 2.25.1
s3fs: None
xarray: 2022.3.0
zarr: 2.11.3
</details>

**Comment by andersy005** [comment, marked as answer]
@yxing6, thank you for your patience


this is not an intake-esm issue per se. The Pangeo-CMIP6 archive is a subset of the CMIP6 data hosted on the ESGF nodes. To request for additional data to be added to the Pangeo-CMIP6 archive, i recommend opening an issue here: https://github.com/pangeo-data/pangeo-cmip6-cloud/issues

**Comment by yxing6** [comment]
Thanks for the updates!
